rotkehlchen/tests/api/test_nfts.py

```diff
@@ -245,18 +245,26 @@ def test_nft_balances_and_prices(rotkehlchen_api_server):
         json={'assets': [NFT_ID_FOR_TEST_ACC4, NFT_ID_FOR_TEST_ACC6_2]},
     )
 
-    # Check that filtering ignored nfts works
-    response = requests.get(api_url_for(
+    # Make sure that ignoring cache doesn't remove any NFTs from the app
+    response_with_cache = requests.get(api_url_for(
         rotkehlchen_api_server,
         'nftsbalanceresource',
     ), json={'async_query': False, 'ignore_cache': False, 'ignored_assets_handling': 'none'})
-    result = assert_proper_response_with_result(response)
-    assert result['entries_found'] == 4
-    assert result['entries_total'] == 4
-    assert result['entries'][TEST_ACC4][0]['id'] == NFT_ID_FOR_TEST_ACC4
-    assert result['entries'][TEST_ACC5][0]['id'] == NFT_ID_FOR_TEST_ACC5
-    assert result['entries'][TEST_ACC6][0]['id'] == NFT_ID_FOR_TEST_ACC6_1
-    assert result['entries'][TEST_ACC6][1]['id'] == NFT_ID_FOR_TEST_ACC6_2
+    response_ignored_cache = requests.get(api_url_for(
+        rotkehlchen_api_server,
+        'nftsbalanceresource',
+    ), json={'async_query': False, 'ignore_cache': True, 'ignored_assets_handling': 'none'})
```

Comment:
you should also test here that with ignored_asset handling on the ignored NFTs **do not** appear.

Reply:
If I understood you correctly, you wanted to check ignored NFTs behaviour when ignoring cache, right? I added this check a  bit lower in this test